mlrun/db/httpdb.py

```diff
@@ -4332,6 +4406,15 @@ def list_alert_templates(self) -> list[mlrun.common.schemas.AlertTemplate]:
             results.append(mlrun.common.schemas.AlertTemplate(**item))
         return results
 
+    @staticmethod
+    def _validate_labels(labels):
```

Comment:
better to have validation on server side to support it for old clients and not restrict 1.8 clients if the validation changes